frontend/src/components/layout/header/UserControls.vue

```diff
@@ -71,54 +85,82 @@
                 icon: '/assets/icons/bell.svg'
             }
         }),
+        created() {
+            this.fetchRelateWebsites();
+        },
         methods: {
             ...mapActions('auth', {
                 logout: LOGOUT
             }),
             ...mapActions('website', {
-                resetData: RESET_DATA
+                resetData: RESET_DATA,
+                changeSelectedWebsite: CHANGE_SELECTED_WEBSITE,
+                fetchRelateWebsites: FETCH_RELATE_WEBSITES
             }),
             endSession() {
                 this.logout();
                 this.resetData();
                 this.$router.push({ name: 'home' });
+            },
+            changeWebsite(data) {
+                let website = {};
+                this.websites.map((item) => {
+                    item.domain === data.value ? website = item : website;
+                });
+                this.changeSelectedWebsite(website);
```

Comment:
Not sure that all this logic should happen here, your action CHANGE_SELECTED_WEBSITE can receive just ID and then you can define selected website based on it inside vuex